### PR TITLE
Remove Kyma reference from readme

### PR DESCRIPTION
**Pull request by derberg**
**Description**

Changes proposed in this pull request:

- Remove Kyma reference from readme. As described in readme it is a generic reusable component that is not only usable in Kyma so we should not impose with the name that this is Kyma dedicated component
- Add reference links to more info about supported specs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Kyma Documentation Component
+# Documentation component
```
Comment (magicmatatjahu) [inline README.md]: .

Comment (mmitoraj) [inline README.md]:
```suggestion
# Documentation Component
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 ## Overview
 
-Kyma Documentation Component (Kyma-DC) is a generic, reusable React component that allows you to render:
+Documentation component is a generic, reusable React component that allows you to render:
```
Comment (mmitoraj) [inline README.md]:
```suggestion
Documentation Component is a generic, reusable React component that allows you to render:
```

File: README.md (hunk 1 above)
Comment (derberg) [reply README.md]: why? component is not part of the name, it is technical indicator that this is a component

**Pull request merged**